### PR TITLE
Portfolio Check 

### PR DESCRIPTION
**Pull request by YogeshGK**
Added portfolio command to check multiple currencies at once.

### CODE REVIEW

**Comment by kaplanmaxe** [comment]
Hey, this is an awesome idea! I really like it. I just added a new script to run eslint. Can you just fix the lint errors? Then I will get this merged in. 

1. Pull master back into your branch
2. `npm install`
3. `npm run lint`

Regarding the conflict in package-lock.json, you can just run `git checkout --theirs package-lock.json`.

Thank you!

File: src/index.js [hunk 1]
```diff
@@ -62,4 +62,57 @@ program
...
+ function showPortfolio(res,obj){
+  console.log('--------------------------------------------------------------------------------------------');
+  console.log(`Symbol   Price     Acct. Val     %Change`);
+  console.log('--------------------------------------------------------------------------------------------');
+  let total = 0;
+  res.forEach(function(currency) {
+
+      let currentVal =  Helpers.round( obj[currency.symbol]*currency.price_usd);
+      total+=Number(currentVal);
+      console.log(`${currency.symbol}    $${Helpers.round(currency.price_usd)}    $${currentVal}      $${currency.percent_change_24h}%`);
```
Comment (kaplanmaxe) [inline src/index.js]: Looks like you have a dollar sign in front of the percentage. Might want to remove that:

`${currency.percent_change_24h}%`

**Comment by kaplanmaxe** [comment]
Thank you! Will publish new update to npm soon

**Comment by YogeshGK** [comment]
Thanks!

**Pull request merged**